scripts/upload_model.py

```diff
@@ -51,6 +55,16 @@ def get_config():
         default=constants.SUBNET_UID,
         help="The subnet UID.",
     )
+    parser.add_argument(
+        "--competition_id",
+        type=CompetitionId,
+        required=True,
```

Comment:
You'll need a similar change to the miner.py file

Reply:
Yeah, I am on it, haven't pushed yet